wrappers/ortools.rb

```diff
@@ -472,7 +476,9 @@ def parse_output(vrp, output)
               elsif activity.type == 'service'
                 collected_indices << current_index
                 service = vrp.services[current_index]
-                point = service.activity&.point || !service.activities.empty? && service.activities[activity.alternative].point
+                service_activity = service.activity ||
+                                   !service.activities.empty? && service.activities[activity.alternative]
```

Comment:
We don't really need `!service.activities.empty? &&`; empty array already returns nil -- i.e., `empty_array = []; empty_array[1] => nil`
```suggestion
                service_activity = service.activity || service.activities[activity.alternative]
```